### ISSUE TITLE
p2pool explanation and sharechain details

### ISSUE THREAD

**Issue by kanzure** [bitcoin-dot-org/bitcoin.org#1168]
Perhaps explaining the sharechain concept and p2pool would be useful for the mining section in the developer guide?

**Comment by harding** [comment]
Good idea.  I actually have some content from an aborted page about mining at https://raw.githubusercontent.com/harding/bitcoin.org/mining/en/mining.md with some images like

![](https://raw.githubusercontent.com/harding/bitcoin.org/mining/img/mining/en-share-chain-block-chain.png)

It may be pretty easy to adapt that to the dev guide.  I'll take a look when I get a chance.  Thanks!

**Comment by ChainQuery** [comment]
This may be helpful as well: 
<img width="741" alt="p2pool_org_-_decentralized_mining_pool__dos-resistant__hop-proof__trust-less" src="https://cloud.githubusercontent.com/assets/13575280/12201358/e349263c-b5f1-11e5-9017-2c556fa73e6e.png">

**Comment by wbnns** [comment]
@harding Heya! I can help add this if you want to connect and let me know what you had in mind.

**Comment by harding** [comment]
@wbnns I was thinking about the section that starts with `#### P2Pool` in the document https://raw.githubusercontent.com/harding/bitcoin.org/mining/en/mining.md along with the related images from that branch.

This could be added to the file `_includes/devdoc/guide_mining.md`

**Comment by wbnns** [comment]
@harding Ok, thank you sir.

**Comment by wbnns** [comment]
[Bounty](https://github.com/bitcoin-dot-org/bitcoin.org#earn-bitcoin-for-contributing): **10,000 bits / ~$20 USD**

**Comment by KingParmenides** [comment]
Submitted a developer docs PR for this bounty: https://github.com/bitcoin-dot-org/developer.bitcoin.org/pull/322

Scope covered:
- P2Pool as a decentralized mining pool design
- share chain structure and cumulative proof of work
- lower-difficulty shares vs Bitcoin-valid blocks
- direct coinbase payout commitments for recent share owners
- decentralization benefits and operational/variance tradeoffs

BTC payout address: 39Q34P8A7g375yqEr8buvNJkUbRgKfbKQZ

**Comment by ryzhkevichpavel-del** [comment]
Submitted an alternative developer docs PR for this bounty: https://github.com/bitcoin-dot-org/developer.bitcoin.org/pull/342

This version follows the maintainer suggestion more closely by expanding the P2Pool/share-chain explanation and including the related diagrams from the older mining guide material referenced in the issue.

If accepted for the bounty, BTC payout can be sent to:

bc1qewlnv3xw3rtll8c5rxjse0fl6v36kkxvwfvccr
